docs/Files/UI/Solution-Manager/Managing-New-Service-Request.md

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
we already have a file for new service requests: Files/UI/Solution-Manager/Creating-Service-Requests

Reply:
Adjusted to use Creating-Service-Requests